benchmarks/view.rb

```diff
@@ -1,24 +1,37 @@
+# frozen_string_literal: true
+
 require 'pathname'
+require 'ostruct'
 require 'benchmark/ips'
-require 'dry/view/renderer'
+require 'dry/view/controller'
 require 'action_view'
 
-class ActionRender
-  include ActionView::Helpers
+TEMPLATES_PATHS = Pathname(__FILE__).dirname.join('templates')
+
+users = [
+  OpenStruct.new(name: 'Tim', link: 'tim@icelab.com`'),
+  OpenStruct.new(name: 'Gustavo', link: 'gustavo@shopify.com`')
```

Comment:
If you don't mind, let's keep personal details out of the example data 😅

Reply:
Oops that was just for me 😄